auto_nag/scripts/configs/tools.json

```diff
@@ -270,6 +270,18 @@
         ],
         "must_run": ["Mon"]
     },
+    "severity_underestimated":{
+        "weeks_lookup": 3,
+        "number_dups": 3,
+        "number_votes": 10,
+        "number_cc": 50,
```

Comment:
We might want to have different thresholds for suggesting moves to S3 or S2 (e.g. 2 duplicates and bug is S4 -> suggest moving to S3, 4 duplicates -> suggest moving to S2).

Wdyt @calixteman?

Reply:
Tracked in #1328

Reply:
I agree.
Is it possible to have some stats about that ? e.g. `number_dups` vs `severity_level` ? ...

Reply:
@calixteman let's move the discussion in the issue (#¹328).